reproduce-figures/README.md

```diff
@@ -1,10 +1,75 @@
+* The script `prepare-scpca-portal-data.R` prepares data obtained from the ScPCA Portal and AWS S3 needed to reproduce figures, tables, and the bulk RNA-Seq analysis.
+Please see instructions below for usage.
   * This script uses functions provided in `utils.R`
-  * This script also uses the `scratch` directory during file processing.
+  * This script also uses the `./scratch` directory during file processing
+* TSV files `FigS1B-gene-expression-data.tsv` and `FigS1C_FigS1D-cell-metrics-data.tsv` are used as input to the script `../scripts/FigS1B-D_method-metrics-comparison.R` which creates Figures S1-D.
+  * Note that this script expects these TSV files to be present in this directory; please do not move them.
+
+**Caution: This process script requires at least 170 GB free storage space.**
 
 ## Instructions
 
-_Forthcoming._
+To reproduce figures and analyses in this repository, you will need to organize and prepare all data needed for input.
+Follow the steps below to do so:
+
+### Step 1: Set up the `renv` environment
+
+This repository uses [`renv`](https://rstudio.github.io/renv/) to manage R packages and dependencies.
+To set up the `renv` environment, open the the `scpca-paper-figures.Rproj` project in RStudio and run `renv::restore()` to install all packages in the provided `renv.lock` file.
+Note that this project uses R version `4.4.0`.
+
+Please refer to the [Introduction to `renv` vignette](https://rstudio.github.io/renv/articles/renv.html) for more information about getting started with `renv`.
+
+### Step 2: Download files from the ScPCA Portal
+
+You will need to download the following files from the ScPCA Portal <https://scpca.alexslemonade.org/>:
+
+1. All ScPCA projects listed in the [Project Whitelist file](../sample-info/project-whitelist.txt)
+  * On each project page (<https://scpca.alexslemonade.org/projects/SCPCPXXXXXX>), download each project with the following options selected:
+    * Ensure the selected modality is `Single-cell`
+    * Ensure the selected Data Format is `SingleCellExperiment (r)`
+    * Do _not_ click to merge all objects into the same sample
+    * If the project contains multiplexed samples, you can exclude those samples (but it will not affect the code if they are included)
+  * These projects will download as zip files; you should leave them in this compressed format, and _do not rename them_.
+
+2. The portal-wide metadata TSV file
+  * You can download this single TSV file using the "Get All Sample Metadata" button on the top-right of the portal homepage.
+
+
+### Step 3: Organize your downloaded files
+
+Place all downloaded ScPCA project files are in the same directory.
+Optionally, you can also store the portal metadata TSV in this directory, but it is not necessary.
+
+### Step 4: Run the `prepare-scpca-portal-data.R` script
+
+Now, you can run the `prepare-scpca-portal-data.R` script specifying the following input arguments:
```

Comment:
I might add the warning here about how much space this script will use?

Reply:
added!